constructor/_schema.py

```diff
@@ -828,6 +829,29 @@ class ConstructorConfiguration(BaseModel):
     Use the standalone binary to perform the uninstallation on Windows.
     Requires conda-standalone 24.11.0 or newer.
     """
+    freeze_base: dict[Literal["conda"], dict] | None = None
+    """
+    Protects the base conda environment against modifications by supported package managers.
+
+    Supported package managers:
+        - `conda`: Protects against conda modifications
+
+    The value for each package manager is a dictionary written to the `frozen` marker file.
+    See CEP-22 for the `frozen` marker file specification.
+
+    Example with empty frozen marker file:
+    ```yaml
+    freeze_base:
+        conda: {}
+    ```
+
+    Example with custom content:
+    ```yaml
+    freeze_base:
+        conda:
+            message: "This base environment is frozen and cannot be modified."
+    ```
```

Comment:
```suggestion
    Supported package managers:
        - `conda`: Protects against conda modifications

    For `conda`, the dictionary is written into the `frozen` marker file.
    See CEP-22 for the `frozen` marker file specification. For example:

    ```
    freeze_base:
        conda:
            message: "This base environment is frozen and cannot be modified."
    ```
```

Some of the information is incorrect. Not every package manager writes into a `frozen` marker file. I still don't think we need to explain to users the different quirks of the YAML syntax - `constructor` is not the place for it.